src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -40,6 +40,8 @@ import aiAnomalyWorkflowFilter from 'images/ai_screenshot-full_anomaly-workflow-
 
 With applied intelligence's anomaly detection, New Relic alerts your team of any unusual behavior instantly. New Relic uses applied intelligence to constantly observe your applications. We use this information to determine your application's baseline, or expected, performance. Whenever behavior deviates from the baseline, we know right away and alert your team so you can address any errors promptly and efficiently. 
 
+There are two types of anomaly detection at New Relic, custom and automatic. Learn about which anomaly detection is right for each situation your team would like to monitor and how to implement anomaly detection in your system. 
```

Comment:
I think there should be a colon instead of a comma after New Relic. The part "is right for each situation your team would like to monitor" reads a bit strange to me, maybe something like "is right for your team and how to implement anomaly detection in your system." ?

Reply:
done